app/templates/main/browse-transferring-body.html

```diff
@@ -118,14 +118,5 @@ <h3 class="govuk-heading-s govuk-heading-s--browse-all-filter">Transfer date</h3
                 </div>
             </div>
         </div>
-        <!-- PAGINATION -->
-        {% if num_records_found > 0 %}
-            <div class="govuk-grid-column-two-thirds browse-grid--two-thirds mobile-pagination">
-                {% with view_name='main.browse_transferring_body', id = request.view_args['_id'] %}
```

Comment:
why have we removed this?

Reply:
the page already has pagination under the table (alike the other pages) this was an extra mention of it that wasn't noticed